src/Lykke.Service.PayAuth/Controllers/VerifyController.cs

```diff
@@ -56,18 +55,12 @@ public class VerifyController : Controller
 
                 return Ok(new SignatureValidationResponse {Description = validationResult.ToString(), ErrorType = validationResult});
             }
-            catch (ClientNotFoundException ex)
+            catch (ClientNotFoundException e)
             {
-                await _log.WriteErrorAsync(nameof(VerifyController), nameof(VerifySignature), request.ToJson(), ex);
+                _log.Error(e, context: request);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.